multi-node/generic/controller-install.sh

```diff
@@ -49,6 +55,12 @@ function init_config {
             exit 1
         fi
     done
+
+    if [ $USE_CALICO = "true" ]; then
+        export K8S_NETWORK_PLUGIN="cni"
```

Comment:
I figured that we shouldn't go overwriting things that the user can set, e.g. as a user I set `K8S_VER=v1.2.4_coreos_cni.0` and enable Calico and then it gets changed back to 1.2.3